db/db_client/db_client_session.go

```diff
@@ -14,11 +14,17 @@ import (
 	"github.com/turbot/steampipe/utils"
 )
 
-func (c *DbClient) AcquireSession(ctx context.Context) *db_common.AcquireSessionResult {
+func (c *DbClient) AcquireSession(ctx context.Context) (asr *db_common.AcquireSessionResult) {
```

Comment:
please do not use acronyms for variable names - `res` will do

Reply:
I am often having to mention this - please remember for the future - no abbreviations/acronyms